### ISSUE TITLE
miscategorization for protocol devs to adopt an alternative client

### ISSUE THREAD

**Issue by adamjonas** [bitcoin-cap/bcap#42]
I'm having trouble understanding this.

"Risk of fragmenting development efforts leading to reduced coordination and increased potential for security vulnerabilities" is a reason for protocol devs to adopt an alternative client?
<img width="844" alt="Screenshot 2024-11-08 at 6 37 53 PM" src="https://github.com/user-attachments/assets/f02202f4-5cd2-4974-a538-32828f7a288f">
<img width="829" alt="Screenshot 2024-11-08 at 6 36 57 PM" src="https://github.com/user-attachments/assets/0f183412-bd04-4e8d-854d-f2d8c6ea127d">
 
This seems incorrect.

**Comment by moneyball** [comment]
Yeah this is just an error. The PDF version has it correct. @rencryptofish can you fix please?

<img width="714" alt="Screenshot 2024-11-08 at 9 22 32 PM" src="https://github.com/user-attachments/assets/dc07ceec-0d96-4398-a4d0-062f5b62ee92">

**Comment by rencryptofish** [comment]
yes i will fix this, thank you for spotting
